src/libcore/yk_swt_impl.c

```diff
@@ -23,6 +23,8 @@ struct mir_loc {
 #define TL_TRACE_REALLOC_CAP    1024
 
 // The trace buffer.
+// This is NULL when the current thread is not tracing, or when a trace has
+// been invalidated due to re-entrance. See below.
```

Comment:
I think there'll be other cases where things are invalidated (e.g. out of memory) so should we just cut "due to re-entrance"?

Reply:
Good idea.